compositor_pipeline/src/pipeline/rtp.rs

```diff
@@ -0,0 +1,62 @@
+use std::net;
+
+use super::Port;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum TransportProtocol {
+    Udp,
+    TcpServer,
+}
+
+pub(super) enum BindToPortError {
+    SocketBind(std::io::Error),
+    PortAlreadyInUse(u16),
+    AllPortsAlreadyInUse { lower_bound: u16, upper_bound: u16 },
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum RequestedPort {
+    Exact(u16),
+    Range((u16, u16)),
+}
```

Comment:
In some places, the range is a tuple, in others it has 2 named fields `lower_bound` and `upper_bound`. Any reasons for that inconsistency?

Reply:
after checking I don't think you are correct. We are using lower_bound and upper_bound only in 2 error types, everywhere else we use tuples to represent range